docs/Documentation/Systems/Vermilion/running.md

```diff
@@ -21,20 +21,29 @@ Vermilion's home directories are shared across all nodes.  There is also /scratc
 
 Partitions are flexible and fluid on Vermilion.  A list of partitions can be found by running the `sinfo` command.  Here are the partitions as of 10/20/2022.
```

Comment:
Also remove vasp from:
"*This page discusses the compute nodes, partitions and gives some examples of building and running applications including running Vasp.*"

Not sure why it's not letting me make that edit.